docs/source/workflows/design_spaces.rst

```diff
@@ -284,3 +284,30 @@ We will require that formulations contain 2 ingredients, that no more than 1 sol
   )
 
   registered_design_space = project.design_spaces.register(design_space)
+
+Sampling from Design Spaces
+---------------------------
+
+To sample candidates from a registered design space, you can use the Citrine Python client as shown in the example below:
```

Comment:
They're already using the Citrine Python client. Just say something like "To sample candidates from a registered design space, follow the example below:"